### ISSUE TITLE
update.sh "copying libssl source" fails not finding sources expected in ssl/Makefile.am 

### ISSUE THREAD

**Issue by BrianInglis** [libressl-portable/portable#309]
autogen.sh runs update.sh which fails "copying libssl source" not finding sources expected in ssl/Makefile.am which should exist under "openbsd/src/lib/libssl/"

$ ./autogen.sh
...
copying libssl source
cp: cannot stat '.../portable/openbsd/src/lib/libssl/s23_clnt.c': No such file or directory
$ fgrep -A4 'copying libssl source' *.sh
update.sh:echo "copying libssl source"
update.sh-rm -f ssl/*.c ssl/*.h
update.sh-for i in `awk '/SOURCES|HEADERS/ { print $3 }' ssl/Makefile.am` ; do
update.sh-      $CP $libssl_src/$i ssl
update.sh-done
$ fgrep 'libssl_src=' update.sh
libssl_src=$CWD/openbsd/src/lib/libssl
$ awk '/SOURCES|HEADERS/ { print "openbsd/src/lib/libssl/"$3 }' ssl/Makefile.am | xargs ls
ls: cannot access 'openbsd/src/lib/libssl/s23_clnt.c': No such file or directory
ls: cannot access 'openbsd/src/lib/libssl/s23_lib.c': No such file or directory
ls: cannot access 'openbsd/src/lib/libssl/s23_pkt.c': No such file or directory
ls: cannot access 'openbsd/src/lib/libssl/s23_srvr.c': No such file or directory
ls: cannot access 'openbsd/src/lib/libssl/s3_both.c': No such file or directory
ls: cannot access 'openbsd/src/lib/libssl/s3_clnt.c': No such file or directory
ls: cannot access 'openbsd/src/lib/libssl/s3_pkt.c': No such file or directory
ls: cannot access 'openbsd/src/lib/libssl/s3_srvr.c': No such file or directory
ls: cannot access 'openbsd/src/lib/libssl/ssl_err2.c': No such file or directory
...

**Comment by busterb** [comment]
I think you left out the most important part from your log, the part where it actually checks out the source code from the openbsd repo. Can you please restore the area where you put 
`...` ?

**Comment by BrianInglis** [comment]
Window long gone - but portable master cloned openbsd master and git pull on both thereafter did nothing, giving the same files as are currently in https://github.com/libressl-portable/openbsd/tree/master/src/lib/libssl/ which does not include the above missing files.
Checking further, looks like https://github.com/libressl-portable/portable/blob/master/ssl/Makefile.am does not show the missing files, so did an explicit git pull origin master from portable, a bunch of files were updated, and Makefile.am now matches directory contents, and autogen.sh completes. 
So it looks like what I had cloned to portable may have had a non-master HEAD? 
Thanks for the prompt to look closer at the repos.
